### ISSUE TITLE
请问用OFA进行ocr训练任务时，如何安排图片、标签文件的层次结构，以及如何使用MsDataset.load语句将图片和对应的标签传递到train_dataset中呢？

### ISSUE THREAD

**Issue by 67766767** [modelscope/modelscope#154]

**Comment by zhangyichang** [comment]
比如有 image_list 和 label_list，可以通过huggingface datasets （pip install datasets) 然后构建python 的内置类型如dict,list或pandas或持久化到csv文件。 
对于list类型，可以用
data_l = [[image_id1, label1, image_file1], [image_id2, label2, image_file2],...]
df = pd.DataFrame(data_l)
df.columns = ['image_id', 'label','image']

dataset = Dataset.from_pandas(df)构建了huggingface dataset
下面代码即可以运行
`import tempfile
from modelscope.msdatasets import MsDataset
from modelscope.metainfo import Trainers
from modelscope.trainers import build_trainer
from modelscope.utils.constant import DownloadMode

train_dataset = MsDataset(dataset)

test_dataset = MsDataset(dataset)

# 可以在代码修改 configuration 的配置
def cfg_modify_fn(cfg):
    cfg.train.hooks = [{
        'type': 'CheckpointHook',
        'interval': 2
    }, {
        'type': 'TextLoggerHook',
        'interval': 1
    }, {
        'type': 'IterTimerHook'
    }]
    cfg.train.max_epochs=2
    return cfg

args = dict(
    model='damo/ofa_ocr-recognition_scene_base_zh',
    model_revision='v1.0.1',
    train_dataset=train_dataset,
    eval_dataset=test_dataset,
    cfg_modify_fn=cfg_modify_fn,
    work_dir = tempfile.TemporaryDirectory().name)
trainer = build_trainer(name=Trainers.ofa, default_args=args)
trainer.train()`

**Comment by balicheng** [comment]
请问image列的image_file1，image_file2是图片路径还是原图片，还是什么样式的图片呢
